modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/OpenAPIResourceTest.java

```diff
@@ -269,20 +301,6 @@ private void _assertObjectRelationshipEndpoints(
 		}
 	}
 
-	private void _assertOpenAPIYAML(
```

Comment:
Maybe you could merge this commit with the previous one. We would be removing useless commits